testplan/common/entity/base.py

```diff
@@ -188,6 +188,17 @@ def all_status(self, target):
             for resource in self._resources
         )
 
+    def _record_resource_exception(self, message, resource, msg_store):
+        fetch_msg = "\n".join(resource.fetch_error_log())
+
+        msg = message.format(
+            resource.cfg.name,
+            traceback.format_exc(),
+            fetch_msg,
```

Comment:
Make these keyword args, so it is visible from the format string what goes where. This is just a Nice To Have